ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__arm64-nightly.yaml

```diff
@@ -393,11 +393,10 @@ tests:
       COMPUTE_NODE_TYPE: m6g.xlarge
       CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
       E2E_RUN_TAGS: '@arm64 and @aws-ipi and @disconnected and @network-ovnkubernetes
-        and @singlenode and not @fips and not @console'
+        and @singlenode and not @fips and not @console and not @storage'
       OCP_ARCH: arm64
       TAG_VERSION: '@4.14'
       TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
-      TEST_SCENARIOS: MCO|ETCD|PSAP|SDN|API_Server|Authentication|Cluster_Operator|Cluster_Infrastructure|OLM|Network_Edge|Operator_SDK|Workloads|Image_Registry|Container_Engine_Tools|NODE|OTA|PerfScale|Cluster_Observability|Security_and_Compliance|LOGGING
```

Comment:
Hmm, needs a bit double confirm why we delete the golang filter, I saw the origin one already remove the `STORAGE`.

Reply:
Sorry for my noise. Yeah, we handle it in case level but these test runs storage may only have 1 or 2 case could passed others are skipped. Maybe we could enhance it later if we have another better way.